### ISSUE TITLE
RFE: skip XDBC server on ML8

### ISSUE THREAD

**Issue by joemfb** [marklogic/roxy#373]
ML8 REST API instances support XCC requests, and do not require a separate XDBC server. Roxy supports this now, by setting `install-xcc=false` and `xcc-port=${app-port}` in `deploy/build.properties` **prior** to bootstrapping. Is there any reasons for these settings to not be the defaults for ML 8 environments?

**Comment by dmcassel** [comment]
+1 on those being defaults

**Comment by paxtonhare** [comment]
Can mlcp specify a different database yet? That was a factor previously that caused me to have to keep installing new XCC instances against my will. I feel like I saw that it could in the latest version.

**Comment by dmcassel** [comment]
Yes, as of MarkLogic 8. http://docs.marklogic.com/guide/relnotes/chap3#id_46078

**Comment by paxtonhare** [comment]
Then +1 for me too.

**Comment by grtjn** [comment]
And if i read correctly, it is against the app specific rest api anyhow.

I'd look forward to it!


Sent from Samsung Mobile

**Comment by grtjn** [comment]
Fix #396 first..

**Comment by grtjn** [comment]
@paxtonhare / @dmcassel: Wondering, would that work for hybrid apps as well, or rest only?

**Comment by dmcassel** [comment]
@grtjn should work fine for hybrid/mvc/bare -- the point is having an XDBC server and the ability to specify where it's loading content. Port 8000 provides that in ML8.

**Comment by grtjn** [comment]
I was just wondering whether the url-rewriter was influencing that, but totally forgot about port 8000.. :)

**Comment by grtjn** [comment]
Depends on #396, postponing it for now..

**Comment by grtjn** [comment]
In retrospect I'm wondering why the database option of MLCP is keeping us back here. We don't use that option now, and can run MLCP happily against app-port since ML8. I'm using that all the time. Actually, we already enforce `xcc-port=${app-port}` with `install-xcc=false` in slush-marklogic-node for quite a while now, and it has been working like a charm.

In short, I see no dependency on #396 (which I think is obsolete too since ML8), particularly if MLCP will just run against app-port, which would be the case with above settings.

Makes sense?
